src/Exceptions/AiClientExceptionInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
```

Comment:
This file should probably live in `Common\Contracts`, or in `Common\Exception\Contracts` - not in its own separate top-level namespace.